src/karmabot/commands/control.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from typing import Dict, List
+
+import karmabot.bot as bot
+from karmabot.slack import get_user_id
+
+
+def join_public_channels(**kwargs):
```

Comment:
valid warning, do we need kwargs?

Reply:
Yes because we generally inject params in all commands. At the moment its `user_id`, `channel` and `text`. Using kwargs allows us to later add or remove injected params without breaking all commands or force us to refactor all commands to match the given params. At the moment its just "here are the kwargs, use them or leave them".

```python
def perform_command(commands, cmd_match, params):
    cmd = cmd_match.group(1)
    try:
        return commands[cmd](**params)
    except KeyError:
        return f"No command '{cmd}' found"
```

Reply:
Ah yes now I remember, so this is a code inspector false positive then.